src/test/java/fi/otavanopisto/kuntaapi/test/server/integration/ptv/TestPtvConsts.java

```diff
@@ -0,0 +1,75 @@
+package fi.otavanopisto.kuntaapi.test.server.integration.ptv;
+
+public class TestPtvConsts {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)